recipes/libtar/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libtar" %}
+{% set version = "1.2.20" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://repo.or.cz/libtar.git/snapshot/0907a9034eaf2a57e8e4a9439f793f3f05d446cd.tar.gz
+  sha256: 4847207d878e79a4acbe32f096e57cd72c9507171953849e4d7eafe312418d95
+
+build:
+  number: 0
+  skip: True  # [win]
+
```

Comment:
Consider adding [run_exports](https://conda-forge.org/docs/meta.html#using-conda-build-s-run-exports-feature) so that downstream libraries can more easily [pin](https://conda-forge.org/docs/meta.html#pinning-packages) the libtar dependency.

You can use the [API/ABI Lab](https://abi-laboratory.pro/index.php?view=tracker) to determine the backwards compatibility libtar

Reply:
This is a good idea!

Reply:
So this would mean just adding
```yaml
  run_exports:
    - libtar
```
to the `build` section, right?

Reply:
Added it. Ready for merging?